internal/cosesign1/cosesign1util_test.go

```diff
@@ -0,0 +1,164 @@
+//go:build linux
+// +build linux
+
+package cosesign1
+
+import (
+	"bytes"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/veraison/go-cose"
+)
+
+func readFileBytes(filename string) ([]byte, error) {
+	content, err := os.ReadFile(filename)
+	if err != nil {
+		println("Error reading '" + filename + "': " + string(err.Error()))
+		return nil, err
+	}
+	if len(content) == 0 {
+		println("Warning: empty file '" + filename + "'")
+	}
+	return content, nil
+}
+
+func readFileString(filename string) (string, error) {
+	data, err := readFileBytes(filename)
+	if err != nil {
+		return "", err
+	}
+	return string(data), nil
+}
+
+func readFileBytesgOrExit(filename string) []byte {
```

Comment:
I think there's a copy/paste error here. Should be "readFileBytesOrExit"?

Reply:
Indeed.